doc/Config_Tutorials/Model_Swaps.md

```diff
@@ -0,0 +1,17 @@
+<!--
+I reccomend viewing this document with the extension Markdown Preview Enhanced (https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced). Once installed to VS Code you can see a html preview by clicking the pane on the right side of VS Code that looks like an open book with a magnifying glass.
+-->
+
+# Model Swaps
+<p>Swapping a model requires two parts which are demonstrated below from our pbo 34thPRC_Overrides for taking the saabr96 and turning it into a m99a2s3 stanchion in appearance. Some code has been removed to focus on just the model swap.</p>

```

Comment:
This is out of date, we no longer do this. We may want to find another example. I think there's a magazine that does this for the bulldog.